src/gas-hub.js

```diff
@@ -33,20 +56,24 @@ $(() => {
           break;
         case 'need relogin':
           initLoginContent();
-          showAlert('Extension has been updated, please relogin', LEVEL_WARN);
+          showLog('Extension has been updated, please relogin', LEVEL_WARN);
           break;
         default:
-          showAlert(err, LEVEL_ERROR);
+          showLog(err, LEVEL_ERROR);
           break;
       }
     });
-});
+}
+
+function unload() {
+  $('.Hu42fb').remove();
+}
+
+
 
 function initContext() {
   context = {};
-  const match = window.location.href.match(/https:\/\/script\.google\.com(.*?)\/home\/projects\/([^/]*)\//);
-  if (!match) return Promise.reject(new Error('not match'));
-  context.id = match[2];
+  const id = getId();
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/083dcd77ddb6869d2b7dc2cdf2f7452c410d4901/src/gas-hub.js#L76"></span>‘id’ is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>